{"name": "Проверка получения заказа с несуществующим трек-номером", "status": "passed", "start": 1712677330063, "stop": 1712677331228, "uuid": "9fd9a307-2eb4-479e-abea-eae44d262ae6", "historyId": "bbcd184ab2b4a9f9d220ed81781c0082", "testCaseId": "bbcd184ab2b4a9f9d220ed81781c0082", "fullName": "tests.orders.test_order_track.TestOrderTrack#test_order_tracking_non_existing_track", "labels": [{"name": "parentSuite", "value": "tests.orders"}, {"name": "suite", "value": "test_order_track"}, {"name": "subSuite", "value": "TestOrderTrack"}, {"name": "host", "value": "DESKTOP-P4QDAQP"}, {"name": "thread", "value": "5944-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.orders.test_order_track"}]}